project/Helpers.scala

```diff
@@ -1,28 +1,28 @@
+import akka.grpc.sbt.AkkaGrpcPlugin
 import com.typesafe.sbt.packager.Keys._
 import com.typesafe.sbt.packager.archetypes.JavaAppPackaging
 import com.typesafe.sbt.packager.docker.DockerPlugin
 import sbt.{Project, Test, Tests, _}
 import sbt.Keys._
 import kalix.sbt.KalixPlugin
+import sbtprotoc.ProtocPlugin.autoImport.PB
+import scalapb.GeneratorOption.{FlatPackage, RetainSourceCodeInfo, SingleLineToProtoString}
 
 /**
  * Contains the versions needed.
  */
 object V {
-  lazy val scala = "2.13.8"
+  lazy val scala = "2.13.10"
```

Comment:
Nope, let's try to go modern with 3.2.2

Reply:
Since 3.2.2 creates errors, we will stay in 2.13.10. Added https://yoppworks.atlassian.net/jira/software/c/projects/IA/issues/IA-126 to eventually resolve this